### ISSUE TITLE
Various bugs, questions and feature requests

### ISSUE THREAD

**Issue by TheSquinter** [Lurkars/gloomhavensecretariat#279]
Have a few bugs, questions and features that came up during CS updates that I thought I'd raise an issue separately for. Sorry for the long post - they've been building up 😆 

- [x] 1. `chooseItem` does not appear to work with -edition

![image](https://user-images.githubusercontent.com/124784924/236208808-591b8ab7-b1f0-4ca2-a99f-bb7a6259af15.png)

```
    "chooseItem": [
        "30-gh",
        "36-gh"
      ]
```

- [x] 2. Card names for all CS monsters show as 'undefined'

![image](https://user-images.githubusercontent.com/124784924/236209439-4492833a-b5ff-4635-b5d7-be7d6b5a9584.png)

- [x] 3. Using `"type": "jump"` without setting a `"value"` shows as "undefined" in monster stat card, but looks fine in ability card

![image](https://user-images.githubusercontent.com/124784924/236210092-ce4af225-66c1-43b3-bd0d-5645c6839ae8.png)

- [x] 4. Also on the terrible-two-archer specifically, they get passive jump to all moves. But it does not appear to add the jump to the abilities. The +1 target works though

![image](https://user-images.githubusercontent.com/124784924/236213074-c1e09f3b-4c3e-4570-906e-e29203262f41.png)

```
      "actions": [
        {
          "type": "target",
          "value": 2,
          "valueType": "add"
        },
        {
          "type": "jump"
        }
      ]
```

- [ ] 5. Can we give the FH Travelling Cloak the +1 HP bonus so it applies on equip? Setting the HP +1 manually is reset after scenario is finished

- [x] 6. Getting the bonus Inspiration from FH rules in CS

![image](https://user-images.githubusercontent.com/124784924/236213942-5008959b-b7ea-44a1-b4b6-002b668e50d7.png)

- [x] 7. Objectives stick around when reducing them to 0 HP

Has this always happened? I usually use the red X 'Remove Objective' button, but noticed they stay there if lowering their HP manually to 0 instead.

![image](https://user-images.githubusercontent.com/124784924/236214706-ea866b59-9496-4e01-b648-dff680e6df1c.png)

- [x] 8. I saw you use the section icon recently. 

 
`<img class=\"icon ghs-svg\" src=\"./assets/images/fh/party/section.svg\">`

Can we add it as a usable icon to turn this:
`"custom": "Read Section #36.7 now"`

![image](https://user-images.githubusercontent.com/124784924/236215080-eb74cff5-ad53-4edc-9dd8-95664621eb90.png)

Into this:

![image](https://user-images.githubusercontent.com/124784924/236215132-2c06edda-9bfd-4042-90a4-0116bc3d9cdc.png)

- [x] 10. Mentioned it in #237, but `calenderSection` rewards seem to be applying 1 week early. They should apply when the box is marked with the X.

- [x] 11. 4 treasures missing from CS Addon scenarios. I added them to the scenario rooms as index 51, 52, 53, 54, just need them added to treasure index file

`"collectiveGold:50"` - don't think collectiveGold can be used in treasures currently though?
`"item:62-gh"` - does edition work with treasures?
`"item:111"`
`"item:110"`

Edit: Added 11

**Comment by Lurkars** [comment]
Thanks a lot, will take deeper look. Short answer to 5.: This should already work! It gets applied after first DRAW in a scenario!

**Comment by Lurkars** [comment]
Phew, a lot to test, but I tried to address every issue in [v0.62.7](https://github.com/Lurkars/gloomhavensecretariat/releases/tag/v0.62.7). So really happy about verifying!

**Comment by Lurkars** [comment]
A short info: I had "exhausted" state on objectives and set this instead of removed. I now differentiate between Objective and Escort on this. So Objective should get removed while Escort should stat at 0 HP with exhausted state.

**Comment by TheSquinter** [comment]
This looks to fix everything! I ended up manually adding the missing treasures for the CS addons, so will be included in next PR.

But it also introduced new issues 😆 

- [ ] The conclusion text and things like `%game.condition.poison%` in treasures now display incorrectly (unsure if there's anything else yet)

![image](https://user-images.githubusercontent.com/124784924/236741450-07c25a89-d073-45ac-8332-6f706801bda0.png)

![image](https://user-images.githubusercontent.com/124784924/236741457-4462f5f3-c0e9-4305-8f7f-e9aa97bec9d4.png)

- [ ] And similar to the jump undefined issue, it looks like Teleport is the same (in CS scenarios at least)

![image](https://user-images.githubusercontent.com/124784924/236741648-e41667c2-a582-43d3-8907-3f4f8ee2a338.png)

**Comment by Lurkars** [comment]
Hopefully fixed this now in [v0.63.3](https://github.com/Lurkars/gloomhavensecretariat/releases/tag/v0.63.3)

**Comment by TheSquinter** [comment]
Great work, all looks fixed
